src/Client.php

```diff
@@ -259,6 +259,17 @@ protected function getForceIpv4Flag(ConnectionInterface $connection)
     }
 
     /**
+     * Extracts the value of the allow-self-signed option from a given connection.
+     *
+     * @param \Phergie\Irc\ConnectionInterface $connection
+     * @return boolean TRUE to allow self signed certificates, FALSE otherwise
+     */
+    protected function getAllowSelfSignedFlag(ConnectionInterface $connection)
+    {
+        return (boolean) $connection->getOption('allow-self-signed') ?: false;
```

Comment:
Not a problem, I followed the existing options as example.
See 'force-ipv4' that should have been 'forceIpv4' ?

If we go for camel case we should update them all, however this breaks api ...

just my 0.02.